CONTRIBUTING.md

```diff
@@ -1,116 +1,132 @@
 
 ### Generate Xtext Artifacts
-Parts of the DOT code base (`org.eclipse.gef.dot`, `org.eclipse.gef.dot.ui`) contain sources that are generated by Xtext. These generated sources are not contained in the Git repository and have to be generated locally within the development workspace. 
+
+Parts of the DOT code base (`org.eclipse.gef.dot`, `org.eclipse.gef.dot.ui`) contain sources that are generated by Xtext. These generated sources are not contained in the Git repository and have to be generated locally within the development workspace.
 
 If you have installed Xtext in your IDE, you can manually trigger the Xtext artifact generation by selecting `Dot.xtext` within the `org.eclipse.gef.dot.internal.language` package in `org.eclipse.gef.dot` in the Package Explorer, and by choosing *Run As -> Generate Xtext Artifacts* from the context menu.
 
 Alternatively you may execute a Maven build locally. Make sure to refresh the *org.eclipse.gef.dot* and *org.eclipse.gef.dot.ui* projects in your workspace after having executed the local build (*Refresh* from context menu on respective projects, or *F5*), and make sure to clean your workspace afterwards (*Project -> Clean -> Clean all projects*).
 
 ## Create and test a contribution
+
 Having properly set up your development environment you can start working on the necessary changes. Please ensure to execute the *all* test-suite that exists for each bundle that was changed to be safe against regressions. In case your change is not trivial, it is recommended to add respective tests to demonstrate that the fix is adequate. You can either call the *AllTests* suite contained in the test bundles of all GEF components, or run a headless Maven build locally (which will also execute the tests).
 
 ## Run a headless build
+
 GEF uses a Maven/Tycho-based build infrastructure. With the [Eclipse Maven Integration](http://eclipse.org/m2e/), a headless build can also be executed in the local workspace. Make sure you have checked out all projects of the respective git repository. Then easily run the build by right-clicking the `pom.xml` file located within the `org.eclipse.gef.releng` project, and selecting 'Run As -> Maven build...'.
 
-In the configuration wizard you have to specify the goals ("clean verify") and the target profile (e.g. "NEON.target") for the build. Note that without setting the target profile the build will not run successfully. The build needs to be executed with a Java 8 VM, so if you have installed multiple JVMs, you should check the settings in the *JRE* tab.
+In the configuration wizard you have to specify the goals ("clean verify") and the target profile (e.g. "2019-03.target") for the build. Note that without setting the target profile the build will not run successfully. The build needs to be executed with a Java 11 VM, so if you have installed multiple JVMs, you should check the settings in the *JRE* tab.
```

Comment:
It seems one has also to define 'JAVAFX_HOME' env variable.
Also the build complains about missing 'javadoc' variable

Reply:
@laeubi: Thanks for the hints and for the feedback!